### ISSUE TITLE
Popover does not reposition when it is in a scrollable parent

### ISSUE THREAD

**Issue by javix** [ng-bootstrap/ng-bootstrap#3927]
### Bug description:
Popover does not reposition when it is in a scrollable parent

### Link to minimally-working StackBlitz that reproduces the issue:
https://angular-jvfaf8.stackblitz.io/

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 10.0.14

ng-bootstrap: 8.0.0

**Comment by Dreadhalor** [comment]
I've found a solution/workaround for this that's working in all the browsers I've tried & it's weirdly simple: `(wheel)="{}"`

That's it. Literally just add a binding to the scroll event on the container that will be scrolled, bind it to absolutely nothing because it serves no purpose other than forcing the DOM to register a scrolling event & suddenly popovers reposition when their parent is scrolled.

I wish I knew how to make a stackblitz for this, but unfortunately I don't. I hope this explanation is simple enough to help anyone else having this issue.
